### PR TITLE
Slot

### PR DESCRIPTION
**Pull request by Yefancy**
#351

### CODE REVIEW

File: src/main/java/gregtech/common/metatileentities/MetaTileEntities.java [hunk 1]
```diff
@@ -717,6 +717,7 @@ public static void init() {
          * - PCM's Ore Addon: 9500-9999
          * - GCM: 10000-10099
          * - MechTech: 10100-10499
+         * - MBT 10500 - 10999
```
Comment (htmlcsjs) [inline src/main/java/gregtech/common/metatileentities/MetaTileEntities.java:720]: hmm i thought we are going to use 32k+ for mte ids for mbt considering thats where i reserved for ct materials

Comment (kumquat-ir) [reply src/main/java/gregtech/common/metatileentities/MetaTileEntities.java:720]: its already reserved on the line directly below this one, this line is redundant

Comment (htmlcsjs) [reply src/main/java/gregtech/common/metatileentities/MetaTileEntities.java:720]: lol
i cant read then

**Review by ALongStringOfNumbers — APPROVED** [review]

File: src/main/java/gregtech/common/metatileentities/MetaTileEntities.java (hunk 1 above)
Comment (Yefancy) [reply src/main/java/gregtech/common/metatileentities/MetaTileEntities.java:720]: This is the ID that MBT needs to extend other mods. (Botania Mana Hatach, Forge Energy Hatach, etc)

**Review by TechLord22 — APPROVED** [review]

**Pull request merged**